File: python/cucumber/model/calculator.py
class Calculator(object):

    def __init__(self, value=0):
        self.result = value

    def reset(self):
        self.result = 0

    def add(self, x, y):
        assert isinstance(x, int) and isinstance(y, int)
        self.result += (x + y)
        return self.result

File: nodejs/cucumber/model/calculator.js
class Calculator {

    constructor(value = 0) {
        this.result = value;
    }

    reset() {
        this.result = 0;
    }

    add(x, y) {
        this.result += (x + y);
        return this.result;
    }
}

module.exports = Calculator;
